server/src/immich/api-v1/asset/asset.controller.ts

```diff
@@ -118,6 +119,38 @@ export class AssetController {
     return responseDto;
   }
 
+  @SharedLinkRoute()
+  @Post('import')
+  @ApiBody({
+    description: 'Asset Import Information',
+    type: ImportAssetDto,
```

Comment:
This API body decorator should not be needed. It can be inferred from the `@Body` type in the method definition, which actually isn't the same type. Is that on purpose?

Reply:
Removed this in next code push.